src/components/Home/Home.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import InfoSpinner from '../InfoSpinner';
+
+const Home: React.FunctionComponent = () => {
+    return <InfoSpinner>Loading Content</InfoSpinner>;
```

Comment:
To reduce confusion, this should probably render something static (perhaps `Welcome to the React Boilerplate!`) so it's clear the page is fully loaded.